stack.yaml

```diff
@@ -14,6 +14,7 @@ extra-deps:
   - paths-0.2.0.0
   - streaming-events-1.0.0
   - strict-tuple-0.1.2
+  - systemd-1.2.0
```

Comment:
@emilypi ran a `stack build` on her Mac with this - and everything compiled fine.